docs/testing.rst

```diff
@@ -263,6 +263,22 @@ This command runs a single python unit test within a specified file.
 
     pytest common/lib/xmodule/xmodule/tests/test_stringify.py::test_stringify
 
+This command runs a single python unit test method within a specified TestCase
+class within a specified file.
+
+::
+
+    pytest common/lib/xmodule/xmodule/tests/test_stringify.py::TestCase::test_stringify
+
+Note: if the method has an `@ddt.data` decorator, ddt will create multiple
+methods with the same prefix name and each individual data input as the suffix
+(e.g. `test_stringify_1_foo`). To test all of the ddt.data variations of the
+same test method, use the pytest `-k` option.
```

Comment:
this is cool.  I'll need to try this.

Reply:
I think the -k option is more convenient than the double-colon syntax above.  Explaining it here makes it seem like something to use only in the case of `@ddt`, but it's much more useful than that.  I would put it above the double-colon syntax, and explain the double-colon as being useful only for disambiguating when there are multiple same-named test methods in different files or classes.

Reply:
I tried rewording this to convey that, let me know if you think it still needs tweaking.